sdcm/nemesis.py

```diff
@@ -1199,6 +1199,7 @@ def _add_and_init_new_cluster_node(self, old_node_ip=None, host_id=None,
             self.log.warning("Node will not be terminated. Please terminate manually!!!")
             raise
         self.cluster.wait_for_nodes_up_and_normal(nodes=[new_node])
+        self.cluster.wait_native_transport(node=new_node)
```

Comment:
need to verify it works also on k8s

Reply:
noting unique here, it's a remote command, and assumes curl is installed (which i'm quite sure is in scylla docker images)